ConsoleApp/Direction.cs

```diff
@@ -0,0 +1,10 @@
+namespace Snake
+{
+    enum Direction
+    {
+        LEFT,
```

Comment:
code convention violated here, should be
```suggestion
        Left,
```